x-pack/plugin/sql/qa/src/main/resources/pivot.csv-spec

```diff
@@ -203,18 +203,4 @@ null           |31070.0
 3              |26830.0        
 4              |24646.0        
 5              |23353.0        
-;
-
-innerAggPivot
```

Comment:
Why did you remove this? Isn't this the case that is supported?

Reply:
https://github.com/elastic/elasticsearch/pull/46489#issuecomment-532769953

Reply:
Thx, I got confused and thought that we'd support this more trivial case.